test_runner/performance/test_bulk_insert.py

```diff
@@ -28,3 +31,38 @@ def test_bulk_insert(neon_with_baseline: PgCompare):
 
```

Comment:
General benchmark question: why do we not wait for the last record lsn to arrive to pageserver?

Flush is just checkpoint + gc. Cannot we be anywhere within the WAL when this flush happens...?

Reply:
Perhaps this explains the 7s vs 2s: the insert does request compaction and then gc, which is probably fast. The recovery does not, but it does wait until the lsn has been reached, but that lsn might not be the same which the bulk_insert ended up at.

Reply:
Good point. But I added a flush and it didn't change the runtime much.

Reply:
> why do we not wait for the last record lsn to arrive to pageserver?

we probably should start doing that as part of the flush method

Reply:
Ok implemented this, and you were right it does explain the big gap :)

Reply:
Should we now close the issue you created? #5257